sql/localsql/localsql.go

```diff
@@ -0,0 +1,59 @@
+package localsql
+
+import (
+	"embed"
+
+	"github.com/spacemeshos/go-spacemesh/sql"
+)
+
+//go:embed schema/schema.sql schema/migrations/*.sql
+var embedded embed.FS
```

Comment:
See `statesql.go` - the same can be done here, by embedding `schema/schema.sql` directly the `sql.LoadSchema` function becomes obsolete.

Reply:
Swiched to string embedding as suggested.
The schema tests I slightly extended and kept under `sql/test`, as it's really undesirable for the `localsql` and `statesql` schema tests to diverge